docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md

```diff
@@ -1,125 +1,98 @@
-
-* [Create network peering for Scalar DL Auditor mode]() // TODO: Create a new document
-
-## Step 9. Prepare a custom values file of Helm Chart of Scalar DL Ledger
-
-You need to configure a custom values file for the Helm Chart of Scalar DL Ledger and Scalar DL Schema Loader (for Ledger) based on your environment (e.g., access information of the database you created in **Step 2**). Please refer to the following document for more details.  
-
-## Step 10. Deploy Scalar DL Ledger using Scalar Helm Chart
-
-Deploy Scalar DL Ledger on your AKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
-
-* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+To make ScalarDL work properly, ScalarDL Ledger and ScalarDL Auditor need to connect with each other. You need to connect two VNets using [Virtual Network Peering](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview). See [Configure network peering for ScalarDL Auditor mode](./NetworkPeeringForScalarDLAuditor.md) fro more details.
 
-## Step 11. Prepare a custom values file of Helm Chart of Scalar DL Auditor
+## Step 9. Prepare a custom values file of Helm Chart of ScalarDL Ledger
 
-You need to configure a custom values file for the Helm Chart of Scalar DL Auditor and Scalar DL Schema Loader (for Auditor) based on your environment (e.g., access information of the database you created in **Step 3**). Please refer to the following document for more details.  
+You need to configure a custom values file for the Helm Chart of ScalarDL Ledger and ScalarDL Schema Loader (for Ledger) based on your environment (e.g., access information of the database you created in **Step 4**). See [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md) for more details.
 
-* [Configure a custom values file of Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+## Step 10. Deploy ScalarDL Ledger using Scalar Helm Chart
 
-## Step 12. Deploy Scalar DL Auditor using Scalar Helm Chart
+Deploy ScalarDL Ledger on your AKS cluster using Scalar Helm Chart. See [Deploy Scalar products using Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/how-to-deploy-scalar-products.md) for more details.
 
-Deploy Scalar DL Auditor on your AKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
+Note: We recommend creating a dedicated namespace by using the `kubectl create ns scalardl-ledger` command and deploying ScalarDL Ledger in the namespace by using the `-n scalardl-ledger` option with the `helm install` command.
```

Comment:
I added a new recommendation based on the offline discussion. We recommend using the dedicated k8s namespace for Scalar product deployment.